From 9c980040da63c9ce08ed9a3b366aa2445b03a0bc Mon Sep 17 00:00:00 2001
From: Fabian Kaczmarczyck <kaczmarczyck@google.com>
Date: Mon, 5 Dec 2022 22:27:43 +0100
Subject: [PATCH] Fixes largeBlob to be consistent with CTAP2

---
 fido2/ctap2/blob.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fido2/ctap2/blob.py b/fido2/ctap2/blob.py
index 03f267f..2363386 100644
--- a/fido2/ctap2/blob.py
+++ b/fido2/ctap2/blob.py
@@ -166,7 +166,7 @@ def write_blob_array(self, blob_array: Sequence[Mapping[int, Any]]) -> None:
             self.ctap.large_blobs(
                 offset,
                 set=_set,
-                length=ln,
+                length=size if offset == 0 else None,
                 pin_uv_protocol=pin_uv_protocol,
                 pin_uv_param=pin_uv_param,
             )
